requirements.txt

```diff
@@ -1,2 +1 @@
-requests==2.22.0
-urllib3==1.24.3
```

Comment:
any reason why this was removed? I think the issue was coming from something related to it